build.sbt

```diff
@@ -26,11 +26,14 @@ scmInfo := Some(
   ScmInfo(url("https://github.com/zio/zio-actors/"), "scm:git:git@github.com:zio/zio-actors.git")
 )
 
-val zioVersion = "1.0.0-RC15"
+val zioVersion    = "1.0.0-RC16"
+val zioNioVersion = "0.3.1"
 libraryDependencies ++= Seq(
-  "dev.zio" %% "zio"          % zioVersion,
-  "dev.zio" %% "zio-test"     % zioVersion % "test",
-  "dev.zio" %% "zio-test-sbt" % zioVersion % "test"
+  "dev.zio"        %% "zio"          % zioVersion,
+  "dev.zio"        %% "zio-nio"      % zioNioVersion,
```

Comment:
So with this introduction, maybe Remoting should be a separate module.

Reply:
Sounds reasonable, so the same as in Akka. I thought about it but I'm not sure how it can be solved - the classes are pretty small and remoting is rooted in `ActorSystem` and `ActorRef` and right now in can't see how to separate them into remote/non-remote classes  (I haven't created modules in sbt yet). 

I see the need of possibility to use actors without remoting therefore remote configuration for `ActorSystem` is optional (if not provided then `ActorSystem` only works as local one).

What do you think about it?

Reply:
I'd like this to be the separate module, but perhaps it's simpler to keep it in the same for this one, and maybe sync with @softinio about extracting it.

Reply:
Sure, so I think for first MVP it can be in one but in future it should be separated.